### PR TITLE
monthly-wechaty-puppet-walnut-report

### PR DESCRIPTION
**Pull request by fabian4**

### CODE REVIEW

File: jekyll/_posts/2022-01-18-monthly-wechaty-puppet-walnut-report.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+- #### [Instance Methods](https://wechaty.js.org/docs/api/message#instance-methods)
+
+  | Instance methods             | Return type         | Support | Detail               |
+  | ---------------------------- | ------------------- | ------- | -------------------- |
+  | from()                       | `Contact` or `null` | ✅       |                      |
+  | to()                         | `Contact` or `null` | ✅       |                      |
+  | room()                       | `Room` or `null`    | ✅       | null                 |
+  | text()                       | `string`            | ✅       |                      |
+  | say(text Or Contact Or File) | `Promise`           | ✅       | ⚠Contact not Support |
```
Comment (su-chang) [inline jekyll/_posts/2022-01-18-monthly-wechaty-puppet-walnut-report.md:82]: Could we convert phone text to Contact Card?

Comment (fabian4) [reply jekyll/_posts/2022-01-18-monthly-wechaty-puppet-walnut-report.md:82]: The 5G platform does not support yet

**Comment by fabian4** [comment]
ping @huan @su-chang @IronConrad

**Review by su-chang — APPROVED** [review]
LGTM

**Review by lijiarui — APPROVED** [review]
LGTM

**Review by huan — APPROVED** [review]
LGTM

Thank you very much for the sharing!

BTW: it seems you are not using the best IDE, I'd like to suggest that you should try VSCode and keep using it.

**Review by IronConrad — APPROVED** [review]
Well done Keep going~

**Pull request merged**